warp10/src/main/java/io/warp10/script/functions/MAP.java

```diff
@@ -215,6 +216,40 @@ private Object applyWithParamsFromMap(WarpScriptStack stack, Map params) throws
           throw new WarpScriptException(getName() + " expects '" + PARAM_OUTPUTTICKS + "' to be list of LONG values.");
         }
       }
+
+      // Check if outputTicks is correctly sorted
+      // In case of a concurrent execution, sorting outputTicks here will lead to a ConcurrentModificationException
+      if (((List<Long>) outputTicks).size() > 1) {
+        if (occurrences < 0) { // reversed
```

Comment:
create a final variable called "reversed" to make the code a bit more readable,
then reuse this variable when calling map